shopfloor_mobile/static/wms/src/components/scenario_picking_detail/picking_summary.js

```diff
@@ -114,53 +116,87 @@ Vue.component("picking-summary-content", {
                     <span class="item-counter">
                         <span>{{ index + 1 }} / {{ count }}</span>
                     </span>
-                    {{ record.title }}
+                    <strong>
+                        {{ options.group_header_title_key ? pkg_type.records[0].product[options.group_header_title_key] : record.title }}
+                    </strong>
+                    <template v-for="(field, index) in options.header_fields">
+                        <component
+                            v-if="field.render_component"
+                            :is="field.render_component"
+                            :options="field.render_options ? field.render_options(record) : {}"
+                            :record="record"
+                            :key="make_component_key([field.render_component, 'list', index, record.id])"
+                        />
+                        <span v-else>
+                            {{ render_field_value(record, field) }}
+                        </span>
+                    </template>
                 </v-expansion-panel-header>
                 <v-expansion-panel-content>
                     <strong class="pkg-type-name mb-2">{{ pkg_type.title }}</strong>
                     <picking-summary-product-detail
                         v-for="(prod, i) in pkg_type.records"
                         :record="prod"
                         :index="i"
+                        :options="options"
                         :key="make_component_key(['pkg', index, i, prod.id])"
                         :count="pkg_type.records.length"
                         />
                 </v-expansion-panel-content>
             </v-expansion-panel>
         </v-expansion-panels>
         <div v-else v-for="(subrec, i) in record.records">
-            <picking-summary-product-detail :record="subrec" :index="index" :count="count" :key="make_component_key(['raw', index, subrec.id, i])" />
+            <picking-summary-product-detail :record="subrec" :index="index" :count="count" :options="options" :key="make_component_key(['raw', index, subrec.id, i])" />
         </div>
     </div>
     `,
 });
 
 Vue.component("picking-summary-product-detail", {
+    mixins: [ItemDetailMixin],
     props: {
         record: Object,
         index: Number,
         count: Number,
+        options: Object,
     },
     template: `
         <div class="summary-content-item">
-            <v-list-item-title>
+            <v-list-item-title v-if="options.show_title">
```

Comment:
this should default to true and is not the case now, no?

Reply:
It does (in the <list> component).